CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+## CHANGELOG
+
+***This document logs all changes to the core code or individual modules which cause a 
```

Comment:
This paragraph doesn't really belong in the file. It's superfluous information for users who are reading it.